The Emerald Sword, an unique and ancient Sheragi warship. (#4786)
endless-sky/endless-sky: data/sheragi outfits.txt

# Copyright (c) 2020 by Fernando S. Sáez
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# Outfits

outfit "Electromagnetic Battery"
	category "Power"
	cost 1000000
	thumbnail "outfit/embattery"
	"mass" 150
	"outfit space" -150
	"shield generation" 0.8
	"shield energy" 0.8
	"shield heat" 0.8
	"energy capacity" 20000

> 	description `This is massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield regeneration of the ship, which is otherwise very underdeveloped.`

Reviewer: I would replace that with "This is a massive superconductive coil, capable of storing large amounts of energy in a magnetic field and discharging all of it in a fraction of a second." The second sentence looks fine.

> 	description `This is massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield regeneration of the ship, which is otherwise very underdeveloped.`

Reviewer: ```suggestion
	description `This is a massive superconductive coil, capable of storing large amounts of energy in a magnetic field and discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield regeneration of the ship, which is otherwise very underdeveloped.`
```

outfit "Fusion Drive"
	category "Engines"
	cost 80000000
	thumbnail "outfit/fusiondrive"
	"mass" 480
	"outfit space" -480
	"engine capacity" -230
	"jump speed" 0.2
	"hyperdrive" 1
	"energy generation" 54
	"heat generation" 156
	"turn" 1920.0
	"turning energy" 3.5
	"turning heat" 9.5
	"thrust" 190
	"thrusting energy" 11
	"thrusting heat" 33
	"reverse thrust" 32
	"reverse thrusting energy" 3.5
	"reverse thrusting heat" 9.5
	"flare sprite" "effect/fdrive"
		"frame rate" 30
		"rewind"
	"flare sound" "atomic huge"
	unplunderable 1

> 	description `The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, with increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but enhanced to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further, and allowing the Emerald Sword to fly at impressive speeds and travel interstellar space with relative ease.`

Reviewer: I would replace the first sentence with "The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, providing increased efficiency at the cost of modularity." The rest looks good to me.

> 	description `The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, with increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but enhanced to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further, and allowing the Emerald Sword to fly at impressive speeds and travel interstellar space with relative ease.`

Reviewer: ```suggestion
	description `The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, providing increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but enhanced to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further and thereby allowing the Emerald Sword to fly at impressive speeds and travel interstellar space with relative ease.`
```

> 	description `The Sheragi engines are much more than simple tubes spitting fire; they are a collection of interdependent devices, providing increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but enhanced to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further and thereby allowing the Emerald Sword to fly at impressive speeds and travel interstellar space with relative ease.`

Reviewer 2: description `The Sheragi engines are much more than simple tubes spewing fire; they are a collection of interdependent devices, providing increased efficiency at the cost of modularity. These include a hyperdrive, an enormous fusion reactor, and the most powerful plasma engines you've ever seen. All simple technology, but honed to their extremes. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further and thereby allowing the Emerald Sword to fly at impressive speeds, travelling through interstellar space with relative ease.`

outfit "Sheragi Cooling Unit"
	category "Systems"
	cost 1000000
	thumbnail "outfit/sheragicooling"
	"mass" 40
	"outfit space" -40
	"active cooling" 142
	"cooling energy" 16.2

> 	description `Sheragi cooling systems are primitive at best, but it is active instead of passive, allowing it to dissipate even more heat than a simple radiator would. It drains a lot of power when the subject to the heat of battle, though.`

Reviewer: ```suggestion
	description `This Sheragi cooling system is primitive at best, but it is active rather than passive, allowing it to dissipate much more heat than a simple radiator would. It drains a lot of power when subjected to the heat of battle, though.`
```

> 	description `This Sheragi cooling system is primitive at best, but it is active rather than passive, allowing it to dissipate much more heat than a simple radiator would. It drains a lot of power when subjected to the heat of battle, though.`

Reviewer 2: description `This Sheragi cooling system is primitive at best, but it works actively rather than passively, allowing it to dissipate much more heat than a simple radiator would. It does, however, drains a lot of power when subjected to the heat of battle.`

Author: English isn't my native language, but wouldn't "does, however, drain" be more gramatically correct?

outfit "Damaged Sheragi Systems"
	category "Special"
	cost 0
	thumbnail "outfit/unknown"
	"shields" -2000
	"hull" -6000
	"energy consumption" 3
	"heat generation" 4
	"jump drive" -1
	unplunderable 1

> 	description "There is a lot of broken Sheragi hardware in here. You need to fix that before you can take the ship back into Human space."

Reviewer 2: description "There is a lot of broken Sheragi hardware in here. You need to fix those before you can take the ship back to Human space."

# Weapons

outfit "Dragonflame Cannon"
	category "Secondary Weapons"
	cost 150000000
	thumbnail "outfit/dragonflame"
	"mass" 156
	"outfit space" -156
	"weapon capacity" -156
	"required crew" 10
	"gun ports" -1
	weapon
		sound "dragonflame" 
		icon "icon/dragonflame"
		"fire effect" "dragonflame"
		"hit effect" "bright hit"
		"hit effect" "dragonflame plasma" 30
		"hit effect" "dragonflame debris" 50
		"velocity" 2000
		"lifetime" 1
		"reload" 180
		cluster
		"firing energy" 16000
		"firing heat" 40000
		"firing force" 1000
		"shield damage" 17000
		"firing fuel" 300
		"hull damage" 9000
		"piercing" 0.3
		"heat damage" 40000
		"hit force" 2000

> 	description "This ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is prohibitive, and it's hard to imagine what could motivate the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, and directs most of the energy of the explosion forwards in a devastintigly powerful beam of gamma radiation."

Reviewer 3: ```suggestion
	description "This ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is prohibitive, and it's hard to imagine what could motivate the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, and directs most of the energy of the explosion forwards in a devastatingly powerful beam of gamma radiation."
```

> 	description "This ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is prohibitive, and it's hard to imagine what could motivate the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, and directs most of the energy of the explosion forwards in a devastatingly powerful beam of gamma radiation."

Reviewer 2: description "The ancient Sheragi superweapon is simple in concept, but the cost involved in developing something like it is exorbitant. It's hard to envision what could have motivated the creation of such a thing. It uses an obscene amount of energy to create an extremely powerful fusion reaction, directing most of the energy of the resulting explosion forwards in a devastatingly powerful beam of gamma radiation."

outfit "Sheragi Anti-Missile"
	category "Turrets"
	cost 845000
	thumbnail "outfit/sheragiam"
	"mass" 35
	"outfit space" -35
	"weapon capacity" -35
	"turret mounts" -1
	weapon
		"hardpoint sprite" "hardpoint/sheragiam"
		"hardpoint offset" 1.5
		"hit effect" "sheragiam"
		"anti-missile" 200
		"velocity" 450
		"lifetime" 1
		"reload" 60
		"firing energy" 220
		"firing heat" 440

> 	description "The sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but takes a long time to recharge. It was probably made to counter nuclear missiles."

Reviewer: ```suggestion
	description "The Sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but takes a long time to recharge. It was probably made to counter nuclear missiles."
```

> 	description "The Sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot, but takes a long time to recharge. It was probably made to counter nuclear missiles."

Reviewer 2: description "This Sheragi anti-missile system fires an extremely powerful laser pulse capable of obliterating any missile in a single shot at the expense of a long recharge time."

outfit "Particle Waveform Turret"
	category "Turrets"
	cost 1650000
	thumbnail "outfit/pwave turret"
	"mass" 47
	"outfit space" -47
	"weapon capacity" -47
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/pwavecannon"
			"frame rate" 20
		sound "pwave"
		"hardpoint sprite" "hardpoint/pwavehp"
		"hardpoint offset" 11
		"hit effect" "pwave impact"
		"inaccuracy" 1.5
		"velocity" 50
		"lifetime" 18
		"reload" 60
		"firing energy" 180
		"firing heat" 651
		"shield damage" 110
		"hull damage" 270
		"heat damage" 1344
		"turret turn" 1.8

> 	description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and easily melts through matter. By the nature of these weapons, one can deduce their tactic was to hit with the Dragonflame Cannon at long range to break the enemy shields and finish them off in relatively closer range with the Particle Waveform Cannon."

Reviewer 2: description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and easily melts through matter. By the nature of these weapons, one can deduce that the tactic used was to use the Dragonflame Cannon at long range to break the enemy shields and finish the ship off at relatively close range with the Particle Waveform Cannon."

#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

# Outfits

outfit "Electromagnetic Battery"
	category "Power"
	cost 1000000
	thumbnail "outfit/embattery"
	"mass" 150
	"outfit space" -150
	"shield generation" 0.8
	"shield energy" 0.8
	"shield heat" 0.8
	"energy capacity" 20000
	description `A massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield system of the ship. Sheragi shield systems seems very underdeveloped otherwise.`

outfit "Fusion Drive"
	category "Engines"
	cost 80000000
	thumbnail "outfit/fusiondrive"
	"mass" 480
	"outfit space" -480
	"engine capacity" -230

> 	"special slot" 2

Reviewer 4: `special slot` sounds very generic. If the goal of this slot is to tie the drive to the ship, then I suggest using a more specific name. I would also expect a "special slot" -1 or -2 somewhere else to make this do something.

It is possible to put all engine stats directly on a ship (effectively making the engine internal to the ship).

Author: Whoops, that is from way back when i had a different idea for the fusion drive's workings.  I'm going to remove that line.

	"reverse thrust" 32
	"reverse thrusting energy" 3.5
	"reverse thrusting heat" 9.5
	"flare sprite" "effect/fdrive"
		"frame rate" 30
		"rewind"
	"flare sound" "atomic huge"
	unplunderable 1
	description `The Sheragi engines are much more than simple tubes spitting fire; They are a collection of interdependent devices, with increased efficiency at the cost of modularity. This includes a hyperdrive, a huge fusion reactor and the most powerful plasma engines you've ever seen. Simple technology, but enhanced to the extreme. The engines are fundamentally connected to the fusion reactor, increasing their thrust even further, and allowing the Emerald Sword to fly at impressive speeds. This piece of engineering is what allowed it to travel interstellar space with relative ease.` 

outfit "Sheragi Cooling Unit"
	category "Systems"
	cost 1000000
	thumbnail "outfit/sheragicooling"
	"mass" 40
	"outfit space" -40
	"active cooling" 142
	"cooling energy" 16.2
	description `Sheragi cooling systems are primitive at best, but it's active, allowing it to dissipate even more heat than a simple radiator would. It drains a lot of power when the subject to the heat of battle, though.`

> outfit "Broken Ship Systems"

Reviewer 4: Creative way to prevent the usage of the jump drive. Maybe change to a less generic name like `Damaged Sheragi Systems`?

	category "Special"
	cost 0
	thumbnail "outfit/unknown"
	"shields" -2000
	"hull" -6000
	"energy consumption" 3
	"heat generation" 4
	#bunks -212
	#"sheragi hardware" 1
	#"required crew" 60
	"jump drive" -1
	unplunderable 1
	description "There is a lot of broken sheragi hardware in here. You need to fix that before you can take the ship back into Human space."

# Weapons

outfit "Dragonflame Cannon"
	category "Secondary Weapons"
	cost 150000000
	thumbnail "outfit/dragonflame"
	"mass" 156
	"outfit space" -156
	"weapon capacity" -156
	"required crew" 10
	"gun ports" -1
	weapon
		sound "dragonflame" 

> 		icon "icon/nuke"

Author: I'm wondering what kind of icon i should use...
